src/averageAges.js

```diff
@@ -1,5 +1,47 @@
 'use strict';
 
+function getSex(people, sex) {
+  return people.filter(person => person.sex === sex);
+}
+
+function getAges(people) {
+  return people.map(person => person.died - person.born);
+}
+
+function getAverageAge(yearsOfLife) {
+  const sumOfAges = yearsOfLife.reduce((sum, age) => sum + age);
+
+  return sumOfAges / yearsOfLife.length;
+}
+
+function getMothers(people) {
+  let arrOfMothersName = new Set(people.map(person => person.mother));
+
+  arrOfMothersName.delete(null);
+  arrOfMothersName = [...arrOfMothersName];
+
+  return people.filter(
+    person => arrOfMothersName.includes(person.name)
+  );
+}
+
+function getChildren(people, sex) {
+  const children = people.filter(person => person.mother !== null);
```

Comment:
```suggestion
  const children = people.filter(person => person.mother);
```
you could simplify it just by writing so